Transactions.md

```diff
@@ -30,9 +30,8 @@ The funding inputs and change output script public keys are negotiated in the of
 
 ## Funding Transaction Input and Output Ordering
 
-The Offerer/Initiator's inputs come first and in the order that they were listed in the Offer message, the Accepter's inputs come after this in the order they are listed in the Accept message. The funding output comes first, followed by the Offerer's change output if applicable, and then the Accepter's change output if applicable.
-
-Note that this will likely change in the future.
+The inputs are sorted by each `funding_input`'s `input_serial_id` in ascending order.
+The outputs are sorted by `change_serial_id` in ascending order with the funding output using the `fund_output_serial_id`.
```

Comment:
suggestion: 
"The outputs are sorted in ascending order based on their respective `change_serial_id` or `fund_output_serial_id`"